src/test/java/com/google/cloud/flink/bigquery/BigQueryArrowFormatTest.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.flink.bigquery;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.flink.api.common.serialization.DeserializationSchema;
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
+import org.apache.flink.table.catalog.CatalogTable;
+import org.apache.flink.table.catalog.CatalogTableImpl;
+import org.apache.flink.table.catalog.ObjectIdentifier;
+import org.apache.flink.table.connector.ChangelogMode;
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.table.descriptors.DescriptorProperties;
+import org.apache.flink.table.factories.DynamicTableFactory.Context;
+import org.apache.flink.table.factories.FactoryUtil;
+import org.apache.flink.table.types.DataType;
+import org.apache.flink.types.RowKind;
+import org.junit.Test;
+
+public class BigQueryArrowFormatTest {
+
+  BigQueryArrowFormat bigQueryArrowFormat =
+      new BigQueryArrowFormat(Arrays.asList("word", "wordcount_output"));
+  MockScanContext mockScanContext = new MockScanContext();
+  static StreamTableEnvironment flinkTableEnv;
+  static String bigqueryReadTable;
+  static String srcQueryString;
+  static String flinkSrcTable;
+  static ObjectIdentifier tableIdentifier;
+  static TableSchema tableSchema;
+  public static final int DEFAULT_PARALLELISM = 10;
+  public static final String FLINK_VERSION = "1.11.0";
+  ImmutableMap<String, String> defaultOptions = ImmutableMap.of("table", "dataset.table");
+
+  public BigQueryArrowFormatTest() {
```

Comment:
why dont you use a @Before clause to initialize things instead of the constructor?